### DISCUSSION TITLE
Ironman Theme by Momentum aka me

### DISCUSSION THREAD

**Discussion by Onslau** [mathoudebine/turing-smart-screen-python#221] [Themes]
![Bild_2023-05-11_221953377](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/69f9f822-5fb3-4f15-9bd7-25c520775f8c)
 link for download here - https://www.mediafire.com/file/0kfbd7180fte293/IRONMAN.rar/file

**Comment by Onslau** [comment]
left under hdd is internet up right is internet Down  middle arc reactor is volume if some not fits then perfect it and send it back here for updated version upload

**Reply by DJRoby19** [reply]
Good great job
But why is the file with a .data extension
Maybe because it's in the original Turing smart format

**Reply by Onslau** [reply]
yea put it in the folder with other themes and ur done

**Reply by Onslau** [reply]
the trick is i use the blanket put in my wallpaper like ironman overly etc  and put things per hand in place thats all
